src/theme.rs

```diff
@@ -1,7 +1,12 @@
 use serde::Deserialize;
-use tui::style::Color;
+use tui::style::{Color, Style};
 
 use self::de::deserialize_option_color_hex_string;
+use crate::THEME;
+
+pub fn style() -> Style {
```

Comment:
Sure we shouldn't ```#[inline]``` these calls? It's called many times under short spans of time it might be costly not to.

Reply:
Yeah wouldn't hurt. I can also try changing `lto = true` for release build and I don't even think this will matter? Let me see what the performance impacts are.

Reply:
Differences are indistinguishable from CPU usage (at least monitoring for a few seconds with top / htop). As in, there may be some difference but without more sophisticated profiling I'm not going to figure it out. However Binary size is much smaller. So maybe I'll enable LTO. 

| Metric | No changes | Inline style() | No inline, LTO = true | Inline & LTO= true |
| - | - | - | - | - |
| CPU Usage | ~2.6% | ~2.6% | ~2.6% | ~2.6% |
| Binary Size | 11MB | 11MB | 7.8MB | 7.8MB |

To err on the side of caution, I'll inline this as well